mu-plugins/blocks/screenshot-preview/postcss/style.pcss

```diff
@@ -26,9 +26,14 @@
 	height: 100%;
 }
 
-.wporg-screenshot--has-error,
-.wporg-screenshot--has-error:hover {
+.wporg-screenshot__has-error,
+.wporg-screenshot__has-error:hover {
 	font-size: 10px;
 	color: #aaa;
 	text-decoration: none;
 }
+
+.wporg-screenshot__loading {
+	color: #aaa;
```

Comment:
This contrast is technically too low, though it's more of a decoration… maybe bump it to `#767676` just to be safe.